src/Text/Pandoc/Writers/ANSI.hs

```diff
@@ -82,7 +82,7 @@ pandocToANSI opts (Pandoc meta blocks) = do
   body <- blockListToANSI opts blocks'
   notes <- gets $ reverse . stNotes
   let notemark x = D.literal (tshow (x :: Int) <> ".") <+> D.space
-  let marks = take (length notes) $ map notemark [1..]
+  let marks = map notemark [1..length notes + 1]
```

Comment:
Is the `+ 1` here right?  I don't see why.  Previously `marks` was a list of the same length as `notes`.  With this change it will be 1 longer than `notes`.

Reply:
Whoops, you're right. Fixed.